web/public/css/dashboard.less

```diff
@@ -62,11 +62,20 @@ label {
     padding: 46px 100px 0 100px;
 }
 
+
+.anchor-jump {
+  padding-top: 140px;
+  margin-top: -140px;
```

Comment:
I had a solution!
```
<h3 class="cd-section-header"><a id="dojos"></a><a href="#dojos">{@i18n key="Member of Dojos"/}</a></h3>
```
```
a[id] {
  display: block;
  position: relative;
  top: -140px;
}
```

The main thing is having two separate anchors for the id and the href. Works a charm.